index.html

```diff
@@ -12,6 +12,17 @@
   <body>
     <header id="store">
       <h1>Greengrocers</h1>
+      <button class="filter-button">Filter</button>
+      <div class="filter-window">
+        <!-- Place for the filter list -->
+      </div>
+      <select name="list-style" id="list-style">
```

Comment:
I love the use of select here to hold all the sorting options, works much better than a button or check boxes.